homu/files/cfg.toml

```diff
@@ -165,10 +165,12 @@ secret = "{{ pillar["homu"]["web-secret"] }}"
         "name": "unicode-script"
     },
     {
-        "name": "webrender"
+        "name": "webrender",
+        "extra_reviewers": [ "jrmuizel", "vvuk" ]
     },
     {
-        "name": "webrender_traits"
+        "name": "webrender_traits",
+        "extra_reviewers": [ "jrmuizel", "vvuk" ]
```

Comment:
This is probably useless because wr-traits now effectively lives in the wr repo.